pages/logo.html

```diff
@@ -8,3 +8,28 @@
 <img class="display-logo" src="{{ site.baseurl }}/assets/img/18F-Logo-M.png" alt="18F logo">
 <br><br>
 <a class="usa-button usa-button-gray" href="{{ site.baseurl }}/assets/dist/18F_Logo.zip">Download logo</a>
+
+<h2>Guidelines</h2>
+<ul>
+	<li>
+		<strong>Do</strong> use the black version of the logo over a white background as your default. Bright blue (#00cfff) and white do not have enough contrast to meet accessibility requirements for low-vision users. Even though logos can be an exception to this rule, we prefer to follow best practices, and use black as our primary logo color. The bright blue version of the logo should only be used if you are using a dark page background. You can see an example of this in our <a href="{{ site.baseurl }}/presentation-themes/">presentation themes</a>.
```

Comment:
@nicoleslaw I made a small adjustment to this paragraph after the discussion in the channel the other day, but please feel free to tinker! Improving this guidance will be an upcoming task: https://github.com/18F/brand/issues/102